docs/mkdocs.yml

```diff
@@ -17,19 +17,29 @@ site_name: Elyra Canvas
 repo_url: https://github.com/elyra-ai/canvas
 docs_dir: pages
 
+# Configuration
 theme:
     name: material
     highlightjs: true
     logo: assets/FlowIcon.png
     favicon: assets/FlowIcon.png
     palette:
         scheme: default
+        primary: black
     features:
       - content.code.copy
       - content.action.edit
       - navigation.tabs
       - navigation.indexes
 
+extra:
+  version:
+    provider: mike
+  social:
+    - icon: fontawesome/brands/github
+      link: http://github.com/elyra-ai/canvas
+      name: Elyra Canvas on GitHub
```

Comment:
The changes above are also edits I made. This shouldn't show as your change.

Reply:
same as above comment.